modules/api/src/main/java/org/apache/ignite/configuration/schemas/network/NetworkConfigurationSchema.java

```diff
@@ -23,18 +23,20 @@
 import org.apache.ignite.configuration.annotation.Value;
 import org.apache.ignite.configuration.storage.ConfigurationType;
 
-/**
- * Configuration schema for network endpoint subtree.
- */
 @ConfigurationRoot(rootName = "network", type = ConfigurationType.LOCAL)
 public class NetworkConfigurationSchema {
-    /** */
+    /** Uniq local node name. */
+    @Value(hasDefault = true)
+    public String name = "";
+
+    /**
+     *
+     */
     @Min(1024)
     @Max(0xFFFF)
     @Value(hasDefault = true)
-    public final int port = 3040;
+    public int port = 47500;
```

Comment:
Why it's not final, are you going to change it in runtime? If true how are you going to handle such changes?

Reply:
Done.